core/metrics/otlp.go

```diff
@@ -28,23 +28,46 @@ func NewOTLPMetricsHandler(buildInfo config.BuildInfo) Handler {
 }
 
 func (n *otlpHandler) Start(ctx context.Context) (err error) {
-	var client otlptrace.Client
-	transport := transportFromString(core.GetEnv(otlpTransportEnv, otlpTransportGRPC.String()))
-	switch transport {
-	case otlpTransportHTTP:
-		client = otlptracehttp.NewClient()
-	case otlpTransportGRPC:
-		client = otlptracegrpc.NewClient()
-	default:
-		return fmt.Errorf("unknown transport type: %s", os.Getenv(otlpTransportEnv))
+	// TODO: generalize this to allow for more than two exporters.
+	client, err := buildClientFromTransport(
+		transportFromString(
+			core.GetEnv(otlpTransportEnv, otlpTransportGRPC.String()),
+		),
+	)
+	if err != nil {
+		return fmt.Errorf("could not create client: %w", err)
+	}
+
+	secondaryClient, err := buildClientFromTransport(
```

Comment:
we should be able to tolerate cases where user doesn't define a secondary transport